src/uct/ib/base/ib_iface.c

```diff
@@ -449,13 +449,13 @@ static ucs_status_t uct_ib_iface_init_lmc(uct_ib_iface_t *iface,
     }
 
     /* count the number of lid_path_bits */
-    num_path_bits = 0;
+    num_path_bits = 1;
```

Comment:
i don't think there's really an overflow here, just needed to fix the assertion in line 492 from `<=` to `<`

Reply:
yes
fixed